chall05/antomart.c

```diff
@@ -0,0 +1,35 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   antomart.c                                         :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: Antomart <Antomart@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2020/04/16 14:52:37 by Antomart          #+#    #+#             */
+/*   Updated: 2020/04/16 14:52:37 by Antomart         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <unistd.h>
+#include <stdio.h>
+ 
+int ft_ie_except_after_c(char* str)
+{
+    int i;
+
+    i = 0;
+    while (str[i])
+    {
+        if (((str[i] == 'i' || str[i] == 'I') && (str[i + 1] == 'e' )) || ((str[i] == 'c' || str[i] == 'C') && (str[i + 1] == 'e') && (str[i + 2] == 'i'))) 
```

Comment:
Why doing some checks with upper case and other without ? Your conditions are very complex for what it is asked you could simplify a lot